src/libs/KeyboardShortcut/index.native.js

```diff
@@ -7,6 +7,9 @@ const KeyboardShortcut = {
         return () => {};
     },
     getDocumentedShortcuts() { return []; },
+    isEnterWhileComposition() {
+        return false;
+    },
```

Comment:
This should be an empty function returned right?

Reply:
If a function is not used in a particular platform `native`, I should not return anything?

Reply:
That's right. It could either be `noop` or if there's a default value. I think `return false` should be fine as in this case, as for native it should be `false` for all the cases.